client/blocks/login/login-form.jsx

```diff
@@ -78,20 +85,21 @@ export class LoginForm extends Component {
 	onSubmitForm = ( event ) => {
 		event.preventDefault();
 
-		const { password, rememberMe, usernameOrEmail } = this.state;
+		const { password, usernameOrEmail } = this.state;
+		let { rememberMe } = this.state;
```

Comment:
See a few lines below with `rememberMe = true;`. If we don't mutate it, then it should be `const`.